src/ResourceManager/Common/Commands.Common.Authentication.ResourceManager/Models/PSAzureEnvironment.cs

```diff
@@ -265,6 +265,7 @@ public string AzureOperationalInsightsEndpoint
         /// Gets or sets the Azure Batch AD resource ID.
         /// </summary>
         public string BatchEndpointResourceId { get; set; }
+        public string StorageOAuthEndpointResourceId { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
@blueww it looks like this line is causing build errors -- would you mind taking a look?